src/Contract/Operation/Associateable.php

```diff
@@ -23,13 +22,10 @@ interface Associateable
      *
      * @see https://loophp-collection.readthedocs.io/en/stable/pages/api.html#associate
      *
-     * @template NewT
-     * @template NewTKey
```

Comment:
similar to the other one, this template param change can be done in a separate PR

Reply:
Yes, fixed !